File: Back_end/shopInlineApi/shopInlineApi/Models/Marca.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace shopInlineApi.Models
{
    public partial class Marca
    {
        public Marca()
        {
            Productos = new HashSet<Producto>();
        }

        public int MarcaId { get; set; }
        public string MarcaDesc { get; set; }
        public DateTime CreateDate { get; set; }
        public DateTime ModifyDate { get; set; }

        public virtual ICollection<Producto> Productos { get; set; }
    }
}

File: src/app/models/marca.ts
export class Marca{
    marcaId: number;
    marcaDesc: string;
    createDate: Date;
    modifyDate: Date;
    constructor(){
        this.marcaId = 0;
        this.marcaDesc ='';
        this.createDate = new Date();
        this.modifyDate = new Date();

    }
}